src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -708,24 +709,12 @@ namespace winrt::TerminalApp::implementation
             {
                 // Possibly update the icon of the tab.
                 page->_UpdateTabIcon(*tab);
-                // Possibly update the title of the tab, window to match the newly
-                // focused pane.
-                page->_UpdateTitle(*tab);
```

Comment:
Double checking. So we don't need this anymore?

Reply:
nope, it was redundant, the title change comes through the `PropertyChanged` handler